site_root/input/resources/examples/ResearchStudy-sd-001.json

```diff
@@ -0,0 +1,44 @@
+{
+  "resourceType": "ResearchStudy",
+  "id": "sd-001",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://fhir.kids-first.io/StructureDefinition/related-organization",
+      "extension": [
```

Comment:
Can you add a `role` example into the `related-organization` extension? Otherwise the unfamiliar observer will wonder why `related-organization` is a complex extension.

Reply:
Updated!